app/src/main/res/values/strings.xml

```diff
@@ -695,4 +695,5 @@
     <string name="sign_out_disconnect">Disconnect</string>
     <!-- Option to cancel signing out shown in confirmation dialog to sign out of account -->
     <string name="sign_out_cancel">Cancel</string>
+    <string name="bookmark_cannot_edit_root">Cannot edit root folders</string>
```

Comment:
We're past string freeze for 1.1, so we'll have to wait until after we cut the release branch to land this.